### PR TITLE
Add node declaration unittests.

### PR DESCRIPTION
**Pull request by olegtropinin**

### CODE REVIEW

File: node/blockchain/tests/fixtures/signed_change_request_message.py [hunk 1]
```diff
@@ -11,3 +12,12 @@ def genesis_signed_change_request_message(primary_validator_key_pair, treasury_a
         treasury_account_number=treasury_account_key_pair.public,
         treasury_amount=treasury_amount
     )
+
+
+@pytest.fixture
+def signed_change_request_message(primary_validator_key_pair, treasury_account_key_pair, treasury_amount):
+    return SignedChangeRequestMessage.create_from(
```
Comment (dmugtasimov) [inline node/blockchain/tests/fixtures/signed_change_request_message.py:19]: is there create_from() method?

File: node/blockchain/tests/fixtures/signed_change_request_message.py [hunk 2]
```diff
@@ -11,3 +12,12 @@ def genesis_signed_change_request_message(primary_validator_key_pair, treasury_a
         treasury_account_number=treasury_account_key_pair.public,
         treasury_amount=treasury_amount
     )
+
+
+@pytest.fixture
+def signed_change_request_message(primary_validator_key_pair, treasury_account_key_pair, treasury_amount):
```
Comment (dmugtasimov) [inline node/blockchain/tests/fixtures/signed_change_request_message.py:18]: is this fixture ever used?

File: node/blockchain/tests/test_models/test_signed_change_request/test_node_declaration.py [hunk 3]
```diff
@@ -24,18 +26,61 @@ def test_create_from_node_declaration_signed_change_request_message(
...
+    node_declaration_signed_change_request_message, regular_node_key_pair
+):
+    signed_change_request = SignedChangeRequest.create_from_signed_change_request_message(
+        message=node_declaration_signed_change_request_message,
+        signing_key=regular_node_key_pair.private,
+    )
+    assert isinstance(signed_change_request, NodeDeclarationSignedChangeRequest)
+    serialized = signed_change_request.json()
+    deserialized = SignedChangeRequest.parse_raw(serialized)
+    assert isinstance(deserialized, SignedChangeRequest)
```
Comment (dmugtasimov) [inline node/blockchain/tests/test_models/test_signed_change_request/test_node_declaration.py:39]: Why not?

`assert isinstance(deserialized, NodeDeclarationSignedChangeRequest)`

File: node/blockchain/tests/test_rest_api/test_signed_change_request/test_node_declaration.py [hunk 4]
```diff
@@ -34,9 +36,34 @@ def test_type_validation_for_node_declaration():
...
+    payload_with_wrong_message_account_lock = deepcopy(payload)
+    payload_with_wrong_message_account_lock['message']['account_lock'] = '0' * 64
+
+    wrong_payloads = [
+        payload_with_wrong_signature,
+        payload_with_wrong_signer,
+        payload_with_wrong_message_account_lock,
+    ]
+
+    for wrong_payload in wrong_payloads:
```
Comment (dmugtasimov) [inline node/blockchain/tests/test_rest_api/test_signed_change_request/test_node_declaration.py:66]: use pytest parametrization instead

File: node/blockchain/tests/fixtures/signed_change_request_message.py (hunk 1 above)
Comment (olegtropinin) [reply node/blockchain/tests/fixtures/signed_change_request_message.py]: I removed this fixture it was not used.

File: node/blockchain/tests/fixtures/signed_change_request_message.py (hunk 2 above)
Comment (olegtropinin) [reply node/blockchain/tests/fixtures/signed_change_request_message.py]: I removed it.

File: node/blockchain/tests/test_models/test_signed_change_request/test_node_declaration.py (hunk 3 above)
Comment (olegtropinin) [reply node/blockchain/tests/test_models/test_signed_change_request/test_node_declaration.py]: I fixed that.

File: node/blockchain/tests/test_rest_api/test_signed_change_request/test_node_declaration.py (hunk 4 above)
Comment (olegtropinin) [reply node/blockchain/tests/test_rest_api/test_signed_change_request/test_node_declaration.py]: Yes, I added pytest parametrization.

**Pull request merged**